ldm/invoke/args.py

```diff
@@ -522,8 +522,8 @@ def _create_arg_parser(self):
             '--ckpt_convert',
             action=argparse.BooleanOptionalAction,
             dest='ckpt_convert',
-            default=False,
-            help='Load legacy ckpt files as diffusers. Pass --no-ckpt-convert to inhibit this behavior',
+            default=True,
+            help='Deprecated option. Legacy ckpt files are now always converted to diffusers when loaded.'
```

Comment:
We are converting legacy .ckpt in memory on every load. If the user wishes to avoid this, they can do a conversion either in the CLI or or the WebUI. It is pretty transparent, except that the .ckpt models load more slowly than they used to.